action.yml

```
@@ -33,6 +33,12 @@ runs:
         toolchain: ${{inputs.toolchain}}
       shell: bash
 
+    - name: Cache Barretenberg transcript
+      uses: actions/cache@v3
+      with:
+        path: ~/noir_cache/ignition/transcript00.dat
+        key: ${{ runner.os }}-nargo-${{steps.parse.outputs.toolchain}}
```

```suggestion
        key: noir_cache
```
This should be platform independent, so we probably only want to cache one copy (the 300mb adds up quickly)